Sources/SwiftTUI/Views/Controls/Button.swift

```diff
@@ -1,60 +1,69 @@
 import Foundation
 
-public struct Button: View, PrimitiveView {
-    public let n: String
+public struct Button<Label>: View, PrimitiveView where Label : View {
+    public let label: Label
```

Comment:
We can make it internal, the SwiftUI Button doesn't expose any properties, guess I didn't notice that before either.